src/pages/accounting/BalancesTable.tsx

```diff
@@ -44,7 +50,7 @@ function BalancesTable(props: IProps) {
                     </TableRow>
                     </TableHead>
                     <TableBody>
-                    {props.peerBalances?.balances.map((peerBalance: PeerBalance, idx: number) => (
+                    {props.peerBalances?.balances.sort((a,b) => b.balance - a.balance).slice(page * 20, page * 20 + 20).map((peerBalance: PeerBalance, idx: number) => (
```

Comment:
I don't think this needs to be paginated. It's not usually more than 100 records, we can do a virtualised tables if performance is the motivation. Either way, if you do want to put this as a PR, the `sort` and `splice` should not be done with each re-render. You can use the `useEffect` which would listen on the `props.peerBalances.balances`.